scripts/opentitan/pyot.py

```diff
@@ -1688,6 +1688,11 @@ def _build_test_args(self, test_name: str) \
         else:
             test_cfg = {k: v for k, v in test_cfg.items()
                         if k not in ('pre', 'post', 'with')}
+            for k, v in test_cfg.items():
```

Comment:
You want to increase the variable names up to 3 char min (I think it is a PEP8 rule).
However, it is fine to keep single letter for variables within a list/dict comprehension.

I'm not sure this works for all cases, i.e. if v is a dict for example (need to double check this, I'll try tomorrow)

Reply:
I don't think this will work if `v` is a dict, you are correct. I was mostly basing this on the aliasing/interpolating functionality in `default`, which I think only checks for `bool` and `list`.

As I'm not as familiar with the script, I'm sure there may be some edge case or nuance that I've missed - please do let me know if you find that I need to add something else here.

Reply:
No worries. Can you provide a sample of the syntax you want to use so I can reproduce the issue and test it? Thanks.

Reply:
I've done a little more testing, I think I'll drop this commit. The problem I was encountering was that when I used the following:
```
{
    aliases: {
        BASEDIR: "${HOME}/path/to/dir"
    }
    testdir: "${BASEDIR}"
    default: {
        rom: "${BASEDIR}/mask_rom_fpga_cw310.elf"
        ... etc. ...
    }
    ...
    tests: {
        kmac_app_rom_test: {
            rom: "${BASEDIR}/mask_rom_fpga_cw310.39.src.vmem"
        }
        ....
    }
 }
 ```
 
 I expected this to work, but instead got errors: `Unable to find ROM file $`
 
 I then changed the interpolation because I thought that was the issue, but then realised that based on the use of `args.rom` in `_build_qemu_fw_args` that `rom` actually needs to be a list, so I actually need the following:
 ```
        kmac_app_rom_test: {
            rom: [
                "${BASEDIR}/mask_rom_fpga_cw310.39.src.vmem"
            ]
        }
 ```
 
 I thought I needed the combination of this and my change but it turns out this works fine without my change, so I guess the interpolation must be happening somewhere else anyhow.